src/Argument.h

```diff
@@ -56,13 +56,11 @@ struct Argument {
      * By default, they are left unset, implying "no default, no min, no max". */
     Expr def, min, max;
 
-  int stride_multiples[4];
```

Comment:
I'm not an expert on this myself. The basic idea is to replay the changes you have made to a branch onto another branch. I think what you want to do is this:

git checkout your_internal_branch
git rebase hexagon  ; hexagon being the github.com/halide/Halide hexagon branch

That *might* just work.

However, you might also want to use git rebase -i hexagon, which will let you interactively replay each commit. If there are redundant changes, this might be useful to skip them. I'm not sure that will be necessary or not. I *think* maybe not.

I gathered this info from reading this page: https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase

Sorry I cannot be more precise with my suggestions. I'd need to try it with your branch to be more sure.